astroid/rebuilder.py

```diff
@@ -104,13 +104,14 @@ def __init__(
             self._parser_module = parser_module
         self._module = self._parser_module.module
 
-    def _get_doc(self, node: T_Doc) -> Tuple[T_Doc, Optional[str]]:
+    def _get_doc(
+        self, node: T_Doc
+    ) -> Tuple[T_Doc, Optional[str], Optional[nodes.Const]]:
```

Comment:
Just picking up on this now:
Why do we return `node` here? Looking at the three instances in which `_get_doc` is called I don't immediately see why we would return an unchanged value.. Do you know this? If not, would you be so kind as to remove it? We could then add a docstring saying that this:
`Returns the docstring of a node both in str and nodes.Const` representation`.

Reply:
done